pandas/tests/io/parser/common/test_common_basic.py

```diff
@@ -862,3 +862,19 @@ def test_read_seek(all_parsers):
             actual = parser.read_csv(file)
         expected = parser.read_csv(StringIO(content))
     tm.assert_frame_equal(actual, expected)
+
+
+def test_pyarrow_with_multi_character_separator():
+    # GH#52554
+    import re
+
+    import pandas as pd
```

Comment:
Isn't this already imported? And if it's not, why not to place both imports at the beginning of the file?

Reply:
Import re is no longer necessary. I moved the import pandas on top.